clang/test/BoundsSafety/Frontend/only_c_is_supported.c

```diff
@@ -0,0 +1,25 @@
+// RUN: not %clang -fbounds-safety-experimental -x c++ %s 2>&1 | FileCheck -check-prefix ERR %s
+
```

Comment:
We use a terse style and use a blank line to group different sets of tests. Here the blank line use is unnecessary and harms readability IMO.

Reply:
@MaskRay Thank you! I removed the blank lines.